categories.json

```diff
@@ -457,75 +457,75 @@
         "shoes": {
           "name": "Shoes",
           "osm_tags": [ { "shop": "shoes" } ],
-          "areas": "all",
+          "-areas": [ "IT" ],
           "form_details": [ "delivery", "drive", "access" ]
         },
         "bag": {
           "name": "Bags",
           "osm_tags": [
             { "shop": "bag" }
           ],
-          "areas": [ "DE", "AT", "CH", "FR", "IT" ],
+          "areas": [ "DE", "AT", "CH", "FR" ],
           "form_details": [ "delivery", "access" ]
         },
         "beauty": {
           "name": "Beauty store",
           "osm_tags": [
             { "shop": "beauty" }
           ],
-          "areas": [ "DE", "AT", "CH", "FR", "IT" ],
+          "areas": [ "DE", "AT", "CH", "FR" ],
           "form_details": [ "delivery", "access" ]
         },
         "child_clothes": {
           "name": "Child clothes",
           "osm_tags": [
             { "shop": "clothes", "clothes": "babies|children" }
           ],
-          "areas": "all",
+          "areas": [ "IT" ],
```

Comment:
Might be `-areas` instead ?

Reply:
It's intended... All shop closed in Italy but babies. You have all clothing shops open.

Reply:
If baby clothes shops are open in Italy, then you can let `"areas": "all"`, this means to display this in all countries including Italy. `"areas": [ "IT" ]` means display __only__ in Italy ;)

Reply:
I know this. But you don't needed it in France.. It is a subset. Anyhow should i change it or can you edit it?

Reply:
OK ! I just understood, sorry for that. So that's good this way